### DISCUSSION TITLE
New to markdoc, how to get started?

### DISCUSSION THREAD

**Discussion by mwala-joel** [markdoc/markdoc#507] [Q&A] [answered]
I've watched the official short reel on markdoc on youtube and i think it wasnt clear enough.[they dont show the folder structure at all and anything and how they are installing react]. The official documantation assumes the developer is already a good one. So how to start with markdoc with and not ruin anything especially if youre planning to document math heavily. Plus if the documentation is purely what the site is all about. Does it mean you can deploy without the backend right. And how?Note: new to react and markdoc itself.

**Comment by mfix-stripe** [comment, marked as answer]
Hey @kazoomarc you can clone this repo and have a Markdoc site up instantly: https://github.com/markdoc/markdoc-starter. It can also be deployed to Vercel from the buttons in the README.

**Reply by mwala-joel** [reply]
Thanks. How to go around the rendering math part and in what element do you recommend like for the math to be inside some div like the call out markdoc tag? Let's say even to add mathjax as the render or anything that can work. It's for math education site with only content.

**Reply by mfix-stripe** [reply]
I'm not familiar with Math libraries, but MathJax looks great at first glance. 

As for how that interfaces with Markdoc, you could create a custom `math` tag and implement the MathJax logic there. 

We have other discussions talking about how to do that: 
1. https://github.com/markdoc/markdoc/discussions/411
2. https://github.com/markdoc/markdoc/discussions/316
